content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -1234,11 +1236,11 @@ contributors. Follow the instructions in the SIG-Windows
 
 * `kubectl port-forward` fails with "unable to do port forwarding: wincat not found"
 
-  This was implemented in Kubernetes 1.15 by including wincat.exe in the
-  pause infrastructure container `mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
-  Be sure to use these versions or newer ones.  If you would like to build your
-  own pause infrastructure container be sure to include
-  [wincat](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/cmd/wincat).
+  Port forwarding support for Windows was requires wincat.exe to be available in the
+  [pause infrastructure container](#pause-image).
+  Ensure you are using a supported image that is compatable with your Windows OS version.
```

Comment:
Should we tell them how or what images are supported per version?

Reply:
maybe as part of the section above where we give the specific version

Reply:
I like @sftim's suggestion to specify what the current recommended pause image is and just update that each release.
All of the past docs for the past releases recommend an image that will work for that release.